.github/workflows/trivy_scan_before_release.yml

```diff
@@ -0,0 +1,25 @@
+name: Scan Antrea Docker image for vulnerabilities before release
+
+on:
+  pull_request:
+    branches:
+      - release-*
+
+jobs:
+  build:
+    if: startsWith(github.event.pull_request.title, 'Release ')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version-file: 'go.mod'
+    - name: Build Antrea Docker image
+      run: |
+        ./hack/build-antrea-linux-all.sh --pull
+    - name: Run Trivy vulnerability scanner on Antrea Docker image
+      uses: aquasecurity/trivy-action@0.8.0
+      with:
+        scan-type: 'image'
+        image-ref: 'antrea/antrea-ubuntu:latest'
+        trivy-config: '.trivy.yml'
```

Comment:
Why it doesn't skip whereabouts like the daily jobs?
Could the daily job use the config file to reduce redundancy?

Reply:
I don't want the daily job to fail because of things we have no direct control over, like whereabouts. However, for the release job, I want us to get an actual picture of all CVEs for the image, even the ones which are not in Antrea binaries.